src/guides/v2.4/graphql/queries/dynamic-blocks.md

```diff
@@ -0,0 +1,116 @@
+---
+group: graphql
+title: dynamicBlocks query
+---
+
+Dynamic blocks can be inserted in CMS pages and blocks using widgets. The `dynamicBlocks` query returns the content of dynamic blocks that implement the Dynamic Blocks Rotator widget.
```

Comment:
we have to make it more obvious that Dynamic Blocks have to be defined in the Dynamic Blocks section of the admin and Dynamic Blocks Rotator is a widget inserted in a CMS page.
Purpose of the new gql query that we added is to grab content of what the widget is returning by parsing some html and extract those parameters that you describe further

That means doing some reference of how the Dynamic Blocks Rotator widget looks like in html when is returned by the cmsPage query.


You can also define widgets

Reply:
I would promote linking to the user guides for how to setup Dynamic Blocks in this section to help provide context. Also note we renamed "Banner" to "Dynamic Block" in 2.3.1 or 2.3.0, so may need to briefly mention that too.